docs/source/teams/overview.rst

```diff
@@ -11,60 +11,82 @@ FiftyOne Teams is designed to integrate naturally into your ML workflow. It inte
 FiftyOne vs FiftyOne Teams
 ___________________________
 
-.. list-table::
-   :widths: 16 7 7 7 7 7 7 7 7 7 7 7 7
-   :header-rows: 1
-   :stub-columns: 1
-
-   * - 
-     - Curate Datasets
-     - Evaluate Models
-     - Find Mistakes
-     - Visualize Embeddings
-     - Deployment
-     - Dataset Management
-     - User Permissions
-     - Dataset Permissions
-     - Dataset Versioning
-     - SSO
-     - Enterprise Support
-     - Licensing
-   * - FiftyOne
-     - ☑
-     - ☑
-     - ☑
-     - ☑
-     - Local, Single user
-     - ☐
-     - ☐
-     - ☐
-     - ☐
-     - ☐
-     - Slack Community
-     - Apache 2.0
-   * - FiftyOne Teams
-     - ☑
-     - ☑
-     - ☑
-     - ☑
-     - | Multi-user, on-premise,
-       | private/public cloud
-     - ☑
-     - ☑
-     - ☑
-     - ☑
-     - ☑
-     - ☑
-     - | Unlimited data, flexible
-       | user-based licensing
+.. raw:: html
```

Comment:
This table is so much nicer! Thanks for improving the look and feel dramatically!